### ISSUE TITLE
Event "input:start" and "input:end" does not provide the color value

### ISSUE THREAD

**Issue by replicat0r** [jaames/iro.js#14]
Hey, 

is it possible to the value registered on these events, just like how it is right now on the color:change  event? this would be incredibly useful to throttle the events to get the final value

**Comment by jaames** [comment]
Hi, sorry for the slow response. (:

I'm not sure if I understand the request properly. You're asking for the color picker's `color` object to be passed to the `input:end` and `input:start` events like it is with `color:set`?, right?

**Comment by replicat0r** [comment]
Hey! No worries. 

What I meant was that there is always a event listner for oncolor change. and the resulting object is a color object, meaning as I am dragging the color wheel around, i have a feed to all the colors. 

It would be good to have the color object available on input:start and input:end, 

so for example, 

("color:change", (color)=>{
  //color object exist here
})

while 

("input:start", (color)=>{
// color doesnt exist here. 
})

I'm not sure what the purpose of having input:start and end if we cant peek into what the value is. 

Basically, If all I cared about is what the end color is, and what the start color is, there is no way to get to it right now.

I hope this helps

**Comment by jaames** [comment]
Ahh, so, the color object that gets passed to the `color:change` callback is just for convenience, you can access it at all times from the color picker's [`color`](https://rakujira.jp/projects/iro/docs/colorPicker_api.html#color) property. :)

e.g:

```js
// create a new color picker
var example = new iro.ColorPicker("#example");

// using the color object from color:change like this:
example.on("color:change", function(color) {
     var hsv = color.hsv;
});

// is the same as this:
example.on("color:change", function() {
     var hsv = example.color.hsv
});

// but the latter works for other events:
example.on("input:start", function() {
     var hsv = example.color.hsv
});
```

I hope that helps! That said, if you still think passing the color into the other method callbacks would be easier, then I'd consider it for sure.
